pages/release-notes.mdx

```diff
@@ -92,14 +92,23 @@ Some of the input and output parameter names of
 [refactor](/advanced-algorithms/available-algorithms/refactor) and
 [schema](/advanced-algorithms/available-algorithms/schema) query modules have
 been changed to align with Memgraph's GraphQL mappings so check if you need to
-change the parameters in your queries. 
+change the parameters in your queries.
 
 Also, some procedures are now functions. Functions can be called inline, while
 procedures are called using the `CALL` clause. Check the documentation for each
-query module for usage examples. 
+query module for usage examples.
 
 </Callout>
 
+Lab v2.10.0 - Dec 6, 2023
+
+{<h3> New features and improvements </h3>}
+
+- Now you can effortlessly query the database using plain English through an
+  OpenAI LLM integration, eliminating the need for Cypher query language.
+- Query summary, statistics, and database notifications will be shown in the summary
+  results tab after each query run.
```

Comment:
```suggestion
- OpenAI LLM integration enables you to query the database using the English language instead of writing Cypher queries.
- After each query run, query summary, statistics, and database notifications are now shown in the result summary
  tab.
```